blogposts/2022/sourcegraph-non-devs.md

```diff
@@ -0,0 +1,211 @@
+
+<p>
+    <iframe src="https://sourcegraph.com/embed/notebooks/Tm90ZWJvb2s6MTkw?theme=light" frameborder="0" sandbox="allow-scripts allow-same-origin allow-popups" style="width: 100%; height: 175px;"></iframe>
+</p>
+
+**Impact:** Our Google Analytics had inconsistent data from historical changes in our web properties and nobody quite knew which web property to use within Google Analytics because they all had different numbers. I did a diff search for which tags were actually on our site and the history of changes and the search above came back with results. I was able to see which tag was in production, [when it was added](https://sourcegraph.com/github.com/sourcegraph/about/-/commit/2b60ef75c86b97bd27c8901c7ff55aadafc55d42), and who to talk to about the history of our Google Tag Manager (spoiler: it was our CEO [Quinn Slack](https://handbook.sourcegraph.com/team/#quinn-slack) – I decided not to bother him with it and just move forward!)
+
+**Time saved:** Exploring Google Analytics with Sourcegraph saved us a couple hours we’d have otherwise spent tracking down who might have historical knowledge of our Google Tag Manager configuration. It likely saved us even more time too, since we needed to investigate past changes in Google Analytics.
+
+### Where do we need to update our contact information and address on our web properties?
+
+When Sourcegraph decided to go [fully remote](https://about.sourcegraph.com/blog/our-journey-to-all-remote/), we changed our permanent office location. Doing this had legal and HR ramifications, so we had to update all our web properties to stay in compliance.
+
+**Search**: [This search](https://sourcegraph.com/search?q=context:%40sourcegraph/all+142+Minna&patternType=literal) helped us update our web properties when we went fully remote.
+
+<p>
+    <iframe src="https://sourcegraph.com/embed/notebooks/Tm90ZWJvb2s6MTkx?theme=light" frameborder="0" sandbox="allow-scripts allow-same-origin allow-popups" style="width: 100%; height: 175px;"></iframe>
+</p>
+
+**Impact:** We were able to maintain consistency among documents and information required by law to keep us out of trouble.
+
+**Time saved:** We saved 10 minutes (and a lot of stress). The alternative would have been searching repo by repo and hoping we found everything.
+
+### Where do fields in HubSpot forms come from and how should we implement new ones?
+
+We use HubSpot forms for all of our opt-in form submissions. These forms are coded into our web properties by the software engineering teams, so it’s essential that the communication about what is being collected is seamless between the marketing, data, and engineering teams.
+
+**Searches**:
+
+We used [this search](https://sourcegraph.com/search?q=context:%40sourcegraph/all+hubspot&patternType=literal) to start the process, then:
```

Comment:
This is the only link here that should runnable. All others should just be links